src/Control/Controller.php

```diff
@@ -693,7 +693,7 @@ public static function normaliseTrailingSlash(string $url): string
         }
 
         // Normlise trailing slash
-        $shouldHaveTrailingSlash = self::config()->uninherited('add_trailing_slash');
+        $shouldHaveTrailingSlash = static::config()->uninherited('add_trailing_slash');
```

Comment:
```suggestion
        $shouldHaveTrailingSlash = Controller::config()->uninherited('add_trailing_slash');
```
This really needs to affect the whole site uniformly. It doesn't make sense for this to change per subclass.

Reply:
As this is a refactoring PR, I'm not changing any existing behaviour. This sort of change should happen in a separate PR, probably to CMS 6

Reply:
This seems like the same sort of change that you did agree to make in https://github.com/silverstripe/silverstripe-cms/pull/2955#discussion_r1642019070? I won't argue though - I can create a new issue for it. Won't be for CMS 6 though, we'd be fixing a bug and there's no API break.

Reply:
OK, given that it would be fixed as a bug I guess makes sense to just just to `Controller::config()` now